app/models/notification.rb

```diff
@@ -15,6 +15,14 @@ def self.unseen
     where(seen_at: nil)
   end
 
+  def self.instantly
+    where(frequency: INSTANTLY)
+  end
+
+  def self.daily
+    where(frequency: DAILY)
+  end
```

Comment:
These are no longer required because of me switching to use an Enum, because that adds them automatically. I'll remove 284501d when I squash this to get rid of them.